volttron/platform/agent/utils.py

```diff
@@ -698,21 +722,34 @@ def process_timestamp(timestamp_string, topic=''):
     return timestamp, original_tz
 
 
-def watch_file(fullpath, callback):
-    """Run callback method whenever the file changes
-
-        Not available on OS X/MacOS.
+def watch_file(path: str, callback: Callable):
```

Comment:
Main change is here where the path was using an event handler that was overly excited.  This is only using the close function